spec/models/user_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  describe '#name_or_email' do
+    subject { user.name_or_email }
+
+    context '名前があるとき' do
+      let(:user) { create(:user) }
```

Comment:
```suggestion
      let(:user) { create(:user, name: 'テスト 太郎') }
```

Factoryの定義を見に行かなくても済むテストコードの方が良いです。

https://speakerdeck.com/willnet/clean-test-code-revised?slide=67

Reply:
なるほど、テストに関わるパラメータはテスト側に書いたほうがわかりやすい、ということですね。
`name` を追加しました。